### PR TITLE
WC-902 Update page-profile to Angular

### PR DESCRIPTION
**Pull request by Klaster1**
https://ggsystems.atlassian.net/browse/WC-902
See subtasks for additional details.

### CODE REVIEW

**Review by AlexDel — APPROVED** [review]
Looks good to me.

File: modules/web-console/frontend/app/app.ts [hunk 1]
```diff
@@ -153,6 +152,13 @@ import igniteServices from './services';
 
 import baseTemplate from 'views/base.pug';
 import * as icons from '../public/images/icons';
+import {igniteWebConsoleAngular} from '../app-angular/downgrade';
+
+import uiRouter from '@uirouter/angularjs';
+import {upgradeModule} from '@uirouter/angular-hybrid';
+// import {downgradeComponent} from '@angular/upgrade/static';
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app/app.ts]: Why it is commented?

File: modules/web-console/frontend/app/app.ts [hunk 2]
```diff
@@ -175,8 +181,9 @@ export default angular
         'ui.grid.resizeColumns',
         'ui.grid.saveState',
         'ui.grid.selection',
-        'ui.router',
-        'ui.router.state.events',
+        uiRouter,
+        upgradeModule.name,
+        // 'ui.router.state.events',
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app/app.ts]: Why  "ui.router.state.events" - commented?

File: modules/web-console/frontend/app/app.ts [hunk 3]
```diff
@@ -253,6 +259,12 @@ export default angular
         noDataCmp.name,
         globalProgressBar.name
     ])
+// Restore routing after Angular loads
+.config(['$urlServiceProvider', ($urlService) => $urlService.deferIntercept()])
+// .directive('serviceBootstrap', downgradeComponent({component: ServiceBootstrapComponent}))
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app/app.ts]: Why next lines commented?

File: modules/web-console/frontend/app-angular/components/form-field/formField.component.scss [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+        &::placeholder {
+            color: rgba(66, 66, 66, 0.5);
+            text-align: left;
+        }
+    }
+    select {
+        -webkit-appearance: unset;
+        appearance: unset;
+        background-image: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAgMCA4IDQiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTAgMGg4bC00IDR6IiBmaWxsPSIjMzkzOTM5Ii8+PC9zdmc+);
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app-angular/components/form-field/formField.component.scss:88]: I would better use normal svg if possible

**Review by akuznetsov-gridgain — APPROVED** [review]
Looks good to me.
Only minor comments from my side.

File: modules/web-console/frontend/app/app.ts (hunk 1 above)
Comment (Klaster1) [reply modules/web-console/frontend/app/app.ts]: Removed.

File: modules/web-console/frontend/app/app.ts (hunk 2 above)
Comment (Klaster1) [reply modules/web-console/frontend/app/app.ts]: Removed, it's no longer used in master either.

File: modules/web-console/frontend/app/app.ts (hunk 3 above)
Comment (Klaster1) [reply modules/web-console/frontend/app/app.ts]: Removed.

File: modules/web-console/frontend/app-angular/components/form-field/formField.component.scss (hunk 4 above)
Comment (Klaster1) [reply modules/web-console/frontend/app-angular/components/form-field/formField.component.scss:88]: It is "normal svg", just inline. I don't really think it matters in this particular case.

**Review by nva — APPROVED** [review]

**Pull request closed without merging**